bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -435,6 +431,35 @@ impl<const N_ARGS: usize> Opcode for CallOpcode<N_ARGS> {
     }
 }
 
+fn write_memory_word_ops(
+    state: &mut CircuitInputStateRef,
+    exec_step: &mut ExecStep,
+    call_id: usize,
+    offset: usize,
+    length: usize,
+    data: &[u8],
+) {
+    let shift = offset % 32;
+    let slot = offset - shift;
+
+    let mut i = 0;
+    while i < length {
```

Comment:
use slot  as address to align memory word.

Reply:
Fix and refactor to copy `execute_precompiled`  result to caller and callee memories in commit [519aacc](https://github.com/scroll-tech/zkevm-circuits/pull/485/commits/519aacca1919ba80492f0bff93a2da5e90267280).